Dmitry_Kononkov/06.Ansible.Workshop/roles/nginx/templates/host.j2

```diff
@@ -0,0 +1,9 @@
+### {{ ansible_managed }}
```

Comment:
Hmmm, I remember this file, I've created this fine on demo for very old group :)